draft-ietf-tls-certificate-compression.md

```diff
@@ -87,7 +87,7 @@ CertificateCompressionAlgorithms value:
     enum {
         zlib(0),
         brotli(1),
-        (255)
+        (65536)
```

Comment:
65535?

Also I just realized this needs to be synced with the table in IANA considerations wrt the zero value (which I missed in my PR), could you do it here as well?